sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/transform/BeamBuiltinAggregations.java

```diff
@@ -453,4 +462,57 @@ public Long extractOutput(Accum accum) {
       return accum.bitAnd;
     }
   }
+
+  public static class LogicalAnd extends CombineFn<Boolean, LogicalAnd.Accum, Boolean> {
```

Comment:
Please add a javadoc comment for this class. You can just use the [ZetaSQL function specification](https://github.com/google/zetasql/blob/master/docs/functions-and-operators.md#logical_and):

"Returns the logical AND of all non-NULL expressions. Returns NULL if there are zero input rows or expression evaluates to NULL for all rows."